### DISCUSSION TITLE
Having issues with authentication

### DISCUSSION THREAD

**Discussion by gkupdx** [roopeshsn/freshbey#78] [General]
![image](https://user-images.githubusercontent.com/32969781/194451240-688db980-3904-42b2-a02f-9b6e81ec2236.png)

![image](https://user-images.githubusercontent.com/32969781/194451032-3b711606-80a2-47ba-bd46-450296c8098d.png)

Hey there, first I just want to say thank you for providing this project as open source. 

I've installed all the dependencies and when I enter "npm run dev", I run into the above issues.
I think it may have something to do with the MongoDB URI. 
I copy+pasted the URI from your README.md and replaced only the password with my database admin password. 

For the Mailtrap credentials, are the credentials supposed to be in quotes or no quotes? For example, 
`EMAIL_USERNAME = "abc123..."` or `EMAIL_USERNAME = abc123...` ?

Any help would be appreciated!

**Comment by gkupdx** [comment]
@roopeshsn hey, sorry for pinging you like this but I'm still kinda stuck on what the problem is here

**Reply by roopeshsn** [reply]
Hi @gkupdx So you've connected your own MongoDB cluster right?

**Reply by roopeshsn** [reply]
In MongoDB URI you should not include these angle brackets <> around your password. Check this once.

**Reply by gkupdx** [reply]
I'm not sure actually. I have my own MongoDB cluster set up but I just copy and pasted the URI in your README.md

And yeah I checked, no angle brackets around the password

**Reply by roopeshsn** [reply]
Have you created the cluster properly?
https://www.mongodb.com/basics/clusters/mongodb-cluster-setup#:~:text=about%20storage%20capacity.-,Creating%20a%20MongoDB%20Cluster,-Depending%20on%20your

**Reply by roopeshsn** [reply]
You have used this URI right? `mongodb+srv://admin:<password>@cluster0.p2u97.mongodb.net/freshbey?retryWrites=true&w=majority`

**Reply by gkupdx** [reply]
yes, and I replaced the <password> with my database password

**Reply by roopeshsn** [reply]
Don't use this URI. Copy the original connection URI from your cluster and use that.

**Reply by gkupdx** [reply]
ohhh wait I think I got it. I connected to localhost:5000 and I'm seeing this:

`{
  message: 'Welcome to Freshbey Backend',
  version: undefined,
  license: 'MIT'
  }`
